### PR TITLE
Data Modeling: Family Relationships

### PR DESCRIPTION
**Pull request by rahul1**

### CODE REVIEW

File: packages/docs/docs/tutorials/data-modeling/family-relationships.md [hunk 1]
```diff
@@ -0,0 +1,278 @@
...
+|  4  | [**Patient**](/docs/api/fhir/resources/patient)/ [**RelatedPerson**](/docs/api/fhir/resources/relatedperson) /[**Person**](/docs/api/fhir/resources/patient) | Yes                                                   | Frequently                                    | All Members                                                                                                    | Individual                                                            |
+|  5  | [**Group**](/docs/api/fhir/resources/group) of Patients                                                                                                      | Yes                                                   | Frequently                                    | All Members                                                                                                    | Family Unit                                                           |
+
```
Comment (reshmakh) [inline packages/docs/docs/tutorials/data-modeling/family-relationships.md:54]: Should give special treatment to minors here, and call out what actions if any they need to take.  Are the minors logging in and viewing their own records?  Getting notified?

File: packages/docs/docs/tutorials/data-modeling/family-relationships.md [hunk 2]
```diff
@@ -0,0 +1,278 @@
...
+- First, you'll have to ask yourself a few questions
+  1. Do family members participate in clinical activiites?
```
Comment (reshmakh) [inline packages/docs/docs/tutorials/data-modeling/family-relationships.md:56]: You should add a note about the CMS queries here - there are many that relate to pediatric use cases.  From a quick skim of the queries it seems like there are maternal screening, PPD and other scenarios that warrant treating parents as patients for reporting purposes https://www.medplum.com/docs/compliance/onc#materials-and-usage

File: packages/docs/docs/tutorials/data-modeling/family-relationships.md [hunk 3]
```diff
@@ -0,0 +1,278 @@
...
+### Do family members participate in clinical activities?
+
+- If the only reason you need to keep track of family members is for their contact information, and your patients generally don't share family members, then you can use the Patient.contact field (see below). This allows you to
+- However, if family members participate in clinical or billing activities, you can reference their participation by creating a [RelatedPerson](/docs/api/fhir/resources/relatedperson) resource for them (see below).
+- Some examples of how a family member may participate in clinical / billing activities:
+  - Child/dependent is the subject of a clinical office visit, parents / family are in attendance
+  - Family member is the insurance subscriber / payor, patient is a spouse or dependent
+  - A Newborn (patient) is administered a medication by a parent
+
+1.  How often will Patients share family members?
```
Comment (reshmakh) [inline packages/docs/docs/tutorials/data-modeling/family-relationships.md:70]: Perhaps explicitly calling this support for siblings might make this more clear

File: packages/docs/docs/tutorials/data-modeling/family-relationships.md [hunk 4]
```diff
@@ -0,0 +1,278 @@
...
+    - For example, two siblings might be enrolled as separate patients in a pediatric clinic, but they share the same mother and father.
+    - In these situations, it is desireable to store demographic and contact information in single resource, to make it easier to update.
+    - For example, you would want a single resource representing both siblings' father, to make sure his information was always up to date.
+    - However, because a [RelatedPerson](/docs/api/fhir/resources/relatedperson) resource models the relationship between a person and a patient, each sibling would need it's own [RelatedPerson](/docs/api/fhir/resources/relatedperson) resource linking them to the father
+    - Therefore, you will need to model the father as either a[Person](/docs/api/fhir/resources/person) or a Patient, and use either the Patient/[RelatedPerson](/docs/api/fhir/resources/relatedperson)/Person or Patient/[RelatedPerson](/docs/api/fhir/resources/relatedperson)/Patient schemes below to model the family.
+
+2.  Which family members will get need clinical information (test results, clinical notes, lab specimens), if any?
```
Comment (reshmakh) [inline packages/docs/docs/tutorials/data-modeling/family-relationships.md:79]: Add reference to neonatal scenarios here, as hearing test, EBP, PPD and many common workflows are linked to this scenario and make it more comprehensible and you have the general case described.

File: packages/docs/docs/tutorials/data-modeling/family-relationships.md [hunk 5]
```diff
@@ -0,0 +1,278 @@
...
+      - Office / Telehealth visit with a physician
+      - Receipt of a medication or prescription
+      - Performance of imaging or labwork
+      - Receipt of a clinical procedure or operation
+    - Each individual who is the focus of a clinical activity should be modeled as a[Person](/docs/api/fhir/resources/patient)resource.
+    - For example:
+      - In pediatric virtual therapy, a single child may be the focus, even if parents are attending sessions
+      - In a maternal care situation, some medications and treatments may be prescribed for the mother, who is the focus of those activities. Additonal therapies might be prescribed for the child, who would be the focus of those activities.
+
+3.  Is the subject of the treatment an indivdual patient, or a family unit?
```
Comment (reshmakh) [inline packages/docs/docs/tutorials/data-modeling/family-relationships.md:92]: Make specific note of Training and Education for family members as this is common for this use case

File: packages/docs/docs/tutorials/data-modeling/family-relationships.md [hunk 6]
```diff
@@ -0,0 +1,278 @@
...
+| [**Person**](/docs/api/fhir/resources/person)               |                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    |
+| [**Group**](/docs/api/fhir/resources/group)                 |                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    |
+
+## Decision Guide
```
Comment (reshmakh) [inline packages/docs/docs/tutorials/data-modeling/family-relationships.md:42]: I'd add another table to make this more comprehensible.  Something like "common in" - newborn/maternal care, adolescent, behavioral health, pediatric specialty care.  Similar to https://www.medplum.com/products/integration#examples-integrations-by-interface-type

**Review by codyebberson — APPROVED** [review]

**Pull request merged**